about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html> 
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> About Alaere </title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+  <nav>
+      <ul class="nav-list">
+        <li><a href="http://127.0.0.1:5500/index.html"> Home </a></li>
+        <li><a href="http://127.0.0.1:5500/about.html"> About </a></li>
+        <li><a href="http://127.0.0.1:5500/portfolio.html"> Portfolio</a></li>
+        <li class="about-header"> About Me!</li>
```

Comment:
This heading isn't part of the navigation links, and would be better placed in a different part of the page. `<nav>` indicates its contents are navigation links, but this part feels more like a page heading, probably the `h1` for the page.